game/state/battle/battleunit.cpp

```diff
@@ -3705,6 +3705,12 @@ void BattleUnit::triggerProximity(GameState &state)
 
 void BattleUnit::triggerBrainsuckers(GameState &state)
 {
+	// Androids do not trigger brainsuckers
+	if(this->agent->type->canTrain == false)
```

Comment:
I don't think keying on if an agent can /train/ makes sense for brainsuckers.

It's probably better to add a new flag, to specifically if a unit triggers brainsuckers? And maybe also in the "does brainsuck succeed" check? (I /seem/ to remember them going for androids in the OG if already hatched, but not sure and not in a position to test right now :)

Reply:
IIRC in OG Brainsuckers will attempt to leap on androids but a chance dictates whether or not they bounce off and live or attempt to suck but melt and always-fail instead

Reply:
I guess you are right. It's move obvious to add new property for brainsucker property. I made some changes. Do I need to edit any other xml files? 
Btw I tested this in the original apoc and the brainsuckers did not even try to go for the android's head. They just stared at him wondering whats going on :)

Reply:
Thanks for quick response Jari
Can you confirm it's the same when the pod is launched at the head?
Im sure ive seen brainsuckers melt on androids somewhere in OG
But then again, my memory is often clouded by Beta2 (which i play more)

Reply:
It's hard to test because they did not shoot any pods at the android. :)  Please check the videos I posted.